We are unable to run code at configure time in a cross environemnt, but as we
control the build we can be fairly certain this dependency is met.

JL 05/07/10
Index: beecrypt-4.1.2/configure.ac
===================================================================
--- beecrypt-4.1.2.orig/configure.ac	2004-12-21 11:37:56.000000000 +0000
+++ beecrypt-4.1.2/configure.ac	2010-07-05 16:09:50.390083494 +0100
@@ -243,32 +243,6 @@
 # Predefines and checks for C++ API support
 AH_TEMPLATE([CPPGLUE],[Define to 1 if you want to include the C++ code])
 
-if test "$ac_with_cplusplus" = yes; then
-  AC_MSG_CHECKING([for IBM's ICU library version >= 2.8])
-  AC_LANG_PUSH(C)
-  AC_RUN_IFELSE([
-    AC_LANG_PROGRAM([[#include <unicode/uversion.h>]],[[
-      #if U_ICU_VERSION_MAJOR_NUM < 2
-      exit(1);
-      #elif U_ICU_VERSION_MAJOR_NUM == 2
-      # if U_ICU_VERSION_MINOR_NUM < 8
-      exit(1);
-      # else
-      exit(0);
-      # endif
-      #else
-      exit(0);
-      #endif
-    ]])],[
-    AC_MSG_RESULT([yes])
-    ],[
-    AC_MSG_RESULT([no])
-    AC_MSG_WARN([disabling cplusplus])
-    ac_with_cplusplus=no
-    ])
-  AC_LANG_POP(C)
-fi
-
 if test "$ac_with_cplusplus" = yes ; then
   AC_DEFINE([CPPGLUE],1)
 fi
